testkit/src/main/scala/org/bitcoins/testkit/core/gen/ScriptGenerators.scala

```diff
@@ -26,7 +27,11 @@ import scala.concurrent.duration.DurationInt
 //TODO: Need to provide generators for [[NonStandardScriptSignature]] and [[NonStandardScriptPubKey]]
 sealed abstract class ScriptGenerators extends BitcoinSLogger {
   val timeout = 5.seconds
-  private val defaultMaxDepth = 2
+  val defaultMaxDepth: Int = 2
+
+  private[gen] def scriptPubKeyTooBig(spk: ScriptPubKey): Boolean = {
+    spk.compactSizeUInt.toLong + CompactSizeUInt(UInt64(520)).bytes.length >= 520L
```

Comment:
This is what this implementation needs to be: 

```scala
  private[gen] def scriptPubKeyTooBig(spk: ScriptPubKey): Boolean = {
    spk.asm.exists(_.bytes.length > ScriptInterpreter.MAX_PUSH_SIZE) && 
      spk.bytes.length == ScriptInterpreter.MAX_SCRIPT_SIZE
  }
```

I think we should just make `MAX_PUSH_SIZE` public, and add a new constant for `MAX_SCRIPT_SIZE = 10000` and refactor the usages in `ScriptInterpreter`